Week2/homework/maartjes-work.js

```diff
@@ -46,18 +46,28 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const result = tasks
+    .map(value => value.duration / 60)
```

Comment:
Instead of using `value` you could also use `task`, to make it more clear you are looking for the duration of a task.